tool/prdeployer.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2017, FaisalAbid. All rights reserved. Use of this source code
```

Comment:
And then please add yourself to the AUTHORS file. Which, if it doesn't exist, please add it :)